crates/iroha_test_network/src/lib.rs

```diff
@@ -207,7 +207,7 @@ impl Network {
         self
     }
 
-    fn topology(&self) -> UniqueVec<PeerId> {
+    fn trusted_peers(&self) -> UniqueVec<Peer> {
```

Comment:
I would prefer to keep this name as `topology` is more general and in my work on #5153 it is used not only as a `sumeragi.trusted_peers`.